benchmarks/benchmarks/stats.py

```diff
@@ -249,6 +249,99 @@ def time_distribution(self, dist_name, method):
         self.method(*self.args, **self.kwds)
 
 
+class TrackContinuousRoundtrip(Benchmark):
+    # Benchmarks that track a value(s) for every distribution can go here
+    param_names = ['dist_name']
+    params = sorted(list(set([d[0] for d in distcont])))
+    dist_data = dict(distcont)
+
+    def setup(self, dist_name):
+        # Distribution setup follows `DistributionsAll` benchmark.
+        # This focuses on ppf, so the code for handling other functions is
+        # removed for simplicity.
+        self.dist = getattr(stats, dist_name)
+
+        if isinstance(self.dist, stats.rv_discrete):
+            raise NotImplementedError("Skipped b/c discrete")
+
```

Comment:
Is this needed since you're only running this on `distcont`?

Reply:
Great point - fixed